### DISCUSSION TITLE
Dogecoin Should be limited

### DISCUSSION THREAD

**Discussion by anujitdatta** [dogecoin/dogecoin#2408] [General]
How crypto market is falling over last couple of months i think doge developers should make limit on dogecoin only then the price will rise otherwise it's hard to cross even 50 cents mark and china banned crypto hitted really hard so please make it happen dogecoin should be limited

**Comment by AbcSxyZ** [comment]
See #2402, similar question was asked a couple of day ago, you should get some answers :)
